modules/statistics/CallStats.js

```diff
@@ -481,55 +608,70 @@ function initCallback(err, msg) {
         return;
     }
 
-    const ret = callStatsBackend.addNewFabric(this.peerconnection,
-        DEFAULT_REMOTE_USER,
-        callStatsBackend.fabricUsage.multiplex,
-        this.confID,
-        this.pcCallback.bind(this));
-
-    const fabricInitialized = ret.status === 'success';
+    // I hate that
+    let atLeastOneFabric = false;
+    let defaultInstance = null;
+
+    for (const callStatsInstance of CallStats.fabrics.values()) {
+        if (!callStatsInstance.hasFabric) {
+            logger.debug('addNewFabric - initCallback');
+            if (callStatsInstance._addNewFabric()) {
+                atLeastOneFabric = true;
+                if (!defaultInstance) {
+                    defaultInstance = callStatsInstance;
+                }
+            }
+        }
+    }
 
-    if (!fabricInitialized) {
-        logger.log('callstats fabric not initilized', ret.message);
+    if (!atLeastOneFabric) {
 
         return;
     }
 
     CallStats.initialized = true;
-    CallStats.feedbackEnabled = true;
+
+    // There is no conference ID nor a PeerConnection available when some of
```

Comment:
I know that the logs needs to be queued, but not sure about the other things. I'm just trying to keep the old behaviour.